.github/workflows/03-preview-url-pr-description.yml

```
@@ -0,0 +1,22 @@
+name: Preview URL PR Description
+
+permissions:
+  pull-requests: write
+  contents: write
```

```suggestion
  pull-requests: write
```
Isn"t pull-requests: write sufficient?